eng/Version.Details.xml

```diff
@@ -297,89 +297,89 @@
       <Uri>https://github.com/dotnet/corefx</Uri>
       <Sha>4ac4c0367003fe3973a3648eb0715ddb0e3bbcea</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.CSharp" Version="5.0.0-alpha1.19516.16" CoherentParentDependency="Microsoft.NETCore.App.Runtime.win-x64">
+    <Dependency Name="Microsoft.CSharp" Version="4.6.0" CoherentParentDependency="Microsoft.NETCore.App.Runtime.win-x64">
```

Comment:
Going back from 5.0.0-alpha1.19516.16 to 4.6.0 does not make sense and shouldn't be happening in an update of AspNetCore-Tooling dependencies. These packages are made coherent with core-setup and EFCore dependencies.

Reply:
Agreed, looks like something is publishing RTM packages to dev channel?  Or subscription here is wrong??? @mmitche  @wtgodbe @anipik

Reply:
I suspect this problem relates to dotnet/arcade#4155 though it looks worse in this repo (which doesn't use System.Diagnostics.DiagnosticSource).

I also suspect an issue with `CoherentParentDependency` attributes because the "Coherency Updates" described above should be tied to EntityFrameworkCore dependencies. Those updates occurred _before_ anything from EntityFrameworkCore came in (see initial 396a4749372cdff207a1a17b16d7491ceba45633 commit). The only 'CoherentParentDependency` attributes in eng/Version.Details.xml tie things to Microsoft.NETCore.App.Runtime.win-x64, Microsoft.Extensions.Logging, and Microsoft.EntityFrameworkCore -- nothing from aspnet/AspNetCore-Tooling.

More specifically, I now suspect the following is messing things up:
``` xml
<Dependency Name="Microsoft.Bcl.AsyncInterfaces" Version="1.0.0" CoherentParentDependency="Microsoft.NETCore.App.Runtime.win-x64">                                                                                                                <Uri>https://github.com/dotnet/corefx</Uri>                                                                             <Sha>4ac4c0367003fe3973a3648eb0715ddb0e3bbcea</Sha>                                                                   </Dependency> 
```
@mmitche shouldn't that dependency be pinned instead of made coherent with core-setup?

> Or subscription here is wrong???

FYI we do not subscribe directly to CoreFx or core-setup dependencies in this repo or AspNetCore-Tooling.
```
> darc get-subscriptions --target-repo aspnet/AspNetCore$ --regex --target-branch master
https://github.com/aspnet/AspNetCore-Tooling (.NET Core 5 Dev) ==> 'https://github.com/aspnet/AspNetCore' ('master')
  - Id: dcfdcc5e-7716-47da-6308-08d6bc51899c
  - Update Frequency: EveryDay
  - Enabled: True
  - Batchable: True
  - Merge Policies:
    AllChecksSuccessful
      ignoreChecks =
                       [
                         "AspNetCore-helix-test",
                         "AspNetCore-helix-test (Tests: Helix ARM64)",
                         "AspNetCore-helix-test (Tests: Helix x64)",
                         "license/cla"
                       ]
    NoRequestedChanges
https://github.com/aspnet/Blazor (.NET Core 5 Dev) ==> 'https://github.com/aspnet/AspNetCore' ('master')
  - Id: db9f3d61-a455-4c33-bf59-08d6ded4953f
  - Update Frequency: EveryDay
  - Enabled: True
  - Batchable: True
  - Merge Policies:
...
https://github.com/aspnet/EntityFrameworkCore (.NET Core 5 Dev) ==> 'https://github.com/aspnet/AspNetCore' ('master')
  - Id: 4cbc2a9e-8529-4b8a-adce-08d67a4e0ac2
  - Update Frequency: EveryDay
  - Enabled: True
  - Batchable: True
  - Merge Policies:
...
https://github.com/dotnet/arcade (.NET Tools - Latest) ==> 'https://github.com/aspnet/AspNetCore' ('master')
  - Id: 0539bdd4-d55d-4517-d708-08d7530b526c
  - Update Frequency: EveryWeek
  - Enabled: True
  - Batchable: True
  - Merge Policies:
...

> darc get-subscriptions --target-repo aspnet/AspNetCore-Tooling --target-branch master
https://github.com/aspnet/Extensions (.NET Core 5 Dev) ==> 'https://github.com/aspnet/AspNetCore-Tooling' ('master')
  - Id: a7bbae07-f7c1-4e4e-fdca-08d6bc6b8555
  - Update Frequency: EveryDay
  - Enabled: True
  - Batchable: True
  - Merge Policies:
    Standard
https://github.com/dotnet/arcade (.NET Tools - Latest) ==> 'https://github.com/aspnet/AspNetCore-Tooling' ('master')
  - Id: d5ce8a70-158f-4712-147d-08d7530b2608
  - Update Frequency: EveryWeek
  - Enabled: True
  - Batchable: True
  - Merge Policies:
    Standard
```

Reply:
There I fixed it https://i.chzbgr.com/full/4136324608/h248BFD26/ aja https://cheezburger.com/4136324608/beeramid-is-locked-in-and-ready-to-go